src/labkey/security/Container.ts

```diff
@@ -291,39 +261,45 @@ export function getHomeContainer(): string {
     return getServerContext().homeContainer;
 }
 
-export interface GetModulesOptions {
+export type GetModulesModules = {
+    /** whether this module should be active for this container */
+    active: boolean
+
+    /** whether this module should be enabled by default for this container */
+    enabled: boolean
+
+    /** Name of the module */
+    name: string
+
+    /** Whether this module is required in the folder type specified above */
+    required: boolean
+
+    /** */
```

Comment:
to be filled in later?

Reply:
Oops, fixed. Really wasn't clear what it means beyond requiring site permissions...